composer.json

```diff
@@ -9,12 +9,11 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.0",
+        "ext-ctype": "*",
         "ext-json": "*",
         "brick/math": "^0.8 || ^0.9",
-        "ramsey/collection": "^1.0",
-        "symfony/polyfill-ctype": "^1.8",
-        "symfony/polyfill-php80": "^1.14"
+        "ramsey/collection": "^1.0"
```

Comment:
@ramsey how do you feel about also dropping this (in a future PR)? As far as I can see, the collection is only used in `BuilderCollection`, but `list<UuidBuilderInterface>` would most likely suffice for the use-case of this library, in my opinion :thinking: 

Obviously to be calibrated and discussed elsewhere, but it's something I noticed while working here.

Reply:
I'll think of a mitigation path in a new PR 👍

Reply:
Resolving here meanwhile, thanks for the input!